docs/alias.md

```diff
@@ -12,24 +12,42 @@ Add a variable alias to the repo configuration
 ```
 USAGE
   $ dvc alias add [--config-path <value>] [--auth-path <value>] [--repo-config-path <value>] [--client-id
-    <value>] [--client-secret <value>] [--project <value>] [--no-api] [--headless] [--caller github|bitbucket|cli]
-    [--alias <value>] [--variable <value>]
+    <value>] [--client-secret <value>] [--project <value>] [--no-api] [--headless] [--caller
```

Comment:
could we take this out to a central page so we it can be managed from one place?

Reply:
not sure how, these docs are auto-generated